larpix/larpix.py

```diff
@@ -928,6 +960,34 @@ def chipid(self, value):
         self.bits[Packet.chipid_bits] = value
 
     @property
+    def cpu_timestamp(self):
+        return self._cpu_timestamp
+
+    @cpu_timestamp.setter
+    def cpu_timestamp(self, value):
+        if not isinstance(value, (int, long, float)):
```

Comment:
I'm not sure it's worth doing this type check, especially because it breaks in Python 3. Suggestions: a) just leave it with no type check, or b) don't even include a setter method.